scipy/constants/__init__.py

```diff
@@ -5,316 +5,104 @@
 
 .. currentmodule:: scipy.constants
 
-Physical and mathematical constants and units.
 
-
-Mathematical constants
-======================
-
-================  =================================================================
-``pi``            Pi
-``golden``        Golden ratio
-``golden_ratio``  Golden ratio
-================  =================================================================
-
-
-Physical constants
+About the SI units
 ==================
+In 2019 seven système international d’unités (SI) base units were (re)defined to
+serve as a basis for all physical quantities by the bureau
+international des poids et mesures (BIPM) [SIunits]_. They can be summarized as followed
```

Comment:
the `[ ]` brackets need removing.

Reply:
Well, here I wanted to provide the link with background information. Should I delete it anyway?

Reply:
No the link is fine, it's just the brackets that are now part of the link and look weird, also `SIunits` isn't the most informative name. How about simply:
```
(see [1])
```